libsigner/src/events.rs

```diff
@@ -0,0 +1,268 @@
+// Copyright (C) 2013-2020 Blockstack PBC, a public benefit corporation
```

Comment:
Would this stacker db communication code be eligible to be moved to the stacks-core SDK that @stjepangolemac is writing?

Reply:
I skimmed through the changes and it seems to me that the way it works is:

1. You run a HTTP server
2. Node sends events to it through a predetermined set of HTTP routes

I'm not sure how you "register" or let the node know where to send the events. Still, the communication looks fairly simple. I suppose we could start with any kind of HTTP server implementation in `stacks-core`.